### PR TITLE
[N/A] Demo app updates - split instruments into 'name' and 'ticker'

### PR DESCRIPTION
**Pull request by pjbroadbent**
Made instruments slightly more "realistic", removed shortcut from teal charts app.

### CODE REVIEW

File: src/demo/components/blotter/SymbolsRow.tsx [hunk 1]
```diff
@@ -83,7 +83,8 @@ export function SymbolsRow(props: SymbolsRowProps): React.ReactElement {
             type: 'fdc3.instrument',
             name: item.name,
             id: {
-                default: item.name
+                ticker: item.ticker,
+                default: item.ticker
```
Comment (tomrobertsOF) [inline src/demo/components/blotter/SymbolsRow.tsx]: If we're changing it to use `ticker` do we still need `default`? That was just a leftover thing from an old version of the spec right? I don't see it on the fdc3 spec site anymore.

**Review by tomrobertsOF — APPROVED** [review]

File: src/demo/apps/BlotterApp.tsx [hunk 2]
```diff
@@ -6,24 +6,25 @@ import {ContextChannelSelector} from '../components/ContextChannelSelector/Conte
...
-    {'name': 'GOOG'},
-    {'name': 'INTC'},
-    {'name': 'MSFT'},
-    {'name': 'NFLX'},
-    {'name': 'NVDA'},
-    {'name': 'PYPL'},
-    {'name': 'QCOM'},
-    {'name': 'ROKU'},
-    {'name': 'TSLA'}
+    {ticker: 'AAPL', name: 'Apple'},
```
Comment (bryangaleopenfin) [inline src/demo/apps/BlotterApp.tsx:13]: "Apple", or "Apple Inc" as elsewhere?

File: src/demo/apps/ChartsApp.tsx [hunk 3]
```diff
@@ -12,11 +12,15 @@ interface AppProps {
 }
 
 export function ChartsApp(props: AppProps): React.ReactElement {
-    const [symbolName, setSymbolName] = React.useState('AAPL');
+    const [symbolName, setSymbolName] = React.useState('Apple Inc (AAPL)');
```
Comment (bryangaleopenfin) [inline src/demo/apps/ChartsApp.tsx]: Think symbolDisplayName would be clearer now (ditto in `NewsApp,tsx`)

File: src/demo/apps/BlotterApp.tsx (hunk 2 above)
Comment (pjbroadbent) [reply src/demo/apps/BlotterApp.tsx:13]: Now normalised, but went with "Apple" - to normalise the other way would mean adding "Inc" to most of the other symbols too, which resulted in some rather long titles and the duplication looked a bit weird in the blotter view.

File: src/demo/apps/ChartsApp.tsx (hunk 3 above)
Comment (pjbroadbent) [reply src/demo/apps/ChartsApp.tsx]: Went with a slightly more generic `title`/`setTitle`

File: src/demo/components/blotter/SymbolsRow.tsx (hunk 1 above)
Comment (pjbroadbent) [reply src/demo/components/blotter/SymbolsRow.tsx]: Hmm, didn't realise that got removed. Yeah, let's do that.

**Review by bryangaleopenfin — APPROVED** [review]

**Pull request merged**